phases/README.md

```diff
@@ -8,22 +8,25 @@
   provides no API stability or versioning. APIs in this directory use
   API module names starting with `wasi_ephemeral_`.
 
-- [`unstable`](unstable): Usable APIs. APIs in `ephemeral` will be
-  occasionally snapshotted and promoted into `unstable`, with approval
+- [`snapshot`](snapshot): Usable APIs. APIs in `ephemeral` will be
+  occasionally snapshotted and promoted into `snapshot`, with approval
   from the Subgroup, considering the overall suitability of the APIs
   themselves, their documentation, test coverage, and availability of
   polyfills when appropriate. Once merged, the API modules will be
   considered stable, though they may be superseded by newer versions.
   Proposals to promote specific APIs should be submitted as Pull Requests
   that:
-   - Move any superseded files out of `unstable` into `old`.
-   - Optionally add polyfills for superseded APIs using `unstable` APIs.
-   - Move all files supporting the APIs out of `ephemeral` into `unstable`.
-   - Rename the API modules from `wasi_ephemeral_` to `wasi_unstable_`
-     and append a version number.
+    1. `git mv` contents of `phases/snapshot/` to `phases/old/snapshot_{old_snapshot_number}`.
+    1. `cp -R` contents of `phases/ephemeral/` into `phases/snapshot/`.
+    1. Rename files copied into `phases/snapshot/` to substitute `ephemeral` for `snapshot` in file names. Append the new snapshot number to each name.
+    1. Update module names given in `.witx` files according to the previous step.
+    1. Update tests in `tools/witx/tests/wasi.rs` to point at new snapshot, and add a test
+       pointing at the just-archived snapshot under `old`.
+    1. Optionally, under `phases/old/snapshot_{old_snapshot_number}, add polyfills for suberseded APIs using the new APIs.
```

Comment:
One way we could add script to do this perhaps?

(I tent to prefer the numbered lists to actually increment the item number as it makes the md for readable/useful,  but I guess that is mostly bikeshed color preference).

Reply:
I worry that creating a script would be fragile because the places where you put the version number are subjective on what files are "top level" (e.g. not typenames), and #98 will make a bunch more top-level modules.

I'll update the markdown, doesnt matter to me either way